taskmanager/composer.json

```diff
@@ -14,5 +14,8 @@
             "App\\": "src/"
         }
     },
-    "license": "MIT"
+    "license": "MIT",
+    "config": {
+        "bin-dir": "bin"
```

Comment:
@tarnawski is that necessary? Without that we need run commands as `vendor/bin/COMMAND` some other pros?

Reply:
Exactly @dominikhajduk. This is a convenient way to expose useful scripts that would otherwise be hidden deep in the `vendor/` directory.